pickcam/build.gradle

```diff
@@ -18,6 +18,15 @@ android {
         versionName version_name
     }
 
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```

Comment:
does it really require toString ? 😮

Reply:
JavaVersion.VERSION_1_8 is an enum
jvmTarget expect string